bindep.txt

```diff
@@ -12,10 +12,12 @@ texlive-luatex85 [platform:rpm]
 texlive-anyfontsize [platform:rpm]
 texlive-ctablestack [platform:rpm]
 texlive-gnu-freefont [platform:rpm]
+texlive-tex-gyre [platform:rpm]
```

Comment:
I could not check what this contains; we don't need the otf versions of the fonts. On Ubuntu, there are indeed in a separate package ``fonts-texgyre``. The ``tex-gyre`` contains the TeX only fonts and LaTeX support files.

Reply:
I found the files on the package for CentOS8:
https://centos.pkgs.org/8/centos-appstream-x86_64/texlive-tex-gyre-20180414-19.el8.noarch.rpm.html

Reply:
Thanks, this contains indeed the needed files (such as tgtermes.sty, t1qtm.fd which are LaTeX support files, and the actual font files in TeX format .tfm or for inclusion in pdf .pfb extension)